development-environment/dev.sh

```diff
@@ -52,7 +52,6 @@ if $dependencies; then
   concurrently "yarn run compose:deps"
   exit 0
 elif $services; then
-  yarn dev:secrets:gen
```

Comment:
Small quality of life improvement so we don't need to always login after dev env restart